polygeist/include/mlir/Dialect/Polygeist/Transforms/Passes.td

```diff
@@ -109,6 +109,7 @@ def SCFCanonicalizeFor : Pass<"canonicalize-scf-for"> {
 
 def LICM : Pass<"licm"> {
   let summary = "Perform LICM on known parallel (and serial) loops";
+  let dependentDialects = ["polygeist::PolygeistDialect"];
```

Comment:
Should we add the `sycl` dialect also here as we will create operations from it?

Reply:
done.